### PR TITLE
feat: seo improvement + fixing link cards

### PR DESCRIPTION
**Pull request by dylansison**
SEO improvements are being made to improve search engine visibility. These include 

- [x] fixing Open Graph images
- [x] adding schema.org search engine metadata
- [x] establishing connections to more highly ranked websites and organizations through indexer-readable vocabulary

The card for shared links stopped functioning properly, perhaps as a result of the move to the domain.

### CODE REVIEW

**Comment by willuhmjs** [comment]
Please do not add us as reviewers until the PR is completed. Additionally, make sure this is labeled as a draft PR since you are still working on it.

**Comment by dylansison** [comment]
ready to be reviewed, mostly done other than accessibility schema tags i could add but don't feel like doing since who knows, someone might just nuke the whole website and start over if we want it to look different
our team stuff and connections still remains relevant tho so...

File: src/app.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 	<head>
 		<meta charset="utf-8" />
-		<link rel="icon" href="%sveltekit.assets%/images/favicon.png" />
+		<link rel="icon" href="https://deepseatactics.com/images/favicon.png" />
```
Comment (tristan-f-r) [inline src/app.html]: This should be %sveltekit.assets% (sveltekit automatically inlines this, and the directory is hashed for caching purposes).

File: src/app.html [hunk 2]
```diff
@@ -30,32 +30,251 @@
...
+				"alternateName": 
+				[
+					"willuhmjs",
+					"willuhm-js"
+				],
+				"mainEntityOfPage": "https://willuhmjs.com",
+				"sameAs": 
+				[
+					"https://github.com/willuhmjs",
+					"https://rusecurectf.radford.edu/users/3421",
```
Comment (tristan-f-r) [inline src/app.html]: This is not a social media account. Preferably, for me, @Daniel-Tomov, and @willuhmjs, leave only our main entity, and delete `sameAs`. Our social media accounts can change at any given period.

File: src/lib/Navbar.svelte [hunk 3]
```diff
@@ -1,4 +1,11 @@
 <nav class="navbar">
+	<script type="application/ld+json">
```
Comment (tristan-f-r) [inline src/lib/Navbar.svelte]: This isn't proper with svelte (and may break the page!), so please remove this.

Even if it was allowed, `cssSelector` is incorrect.

File: src/app.html
Comment (tristan-f-r) [inline src/app.html:1]: In general, the json-ld schemas from this webpage should be instead typechecked and reserialized in `+layout.svelte` (see [reference article](https://navillus.dev/blog/json-ld-in-sveltekit)).

**Review by tristan-f-r — CHANGES_REQUESTED** [review]
I'll be formatting this PR once my reviews are resolved, to help clear some broken formatting mistakes.

Given that you have no prior experience with Svelte, do let me know if you want me to handle moving the meta tags to a type-checked system.

File: src/app.html (hunk 2 above)
Comment (willuhmjs) [reply src/app.html]: I agree

File: src/app.html (hunk 1 above)
Comment (dylansison) [reply src/app.html]: done

File: src/app.html (hunk 2 above)
Comment (dylansison) [reply src/app.html]: done

File: src/lib/Navbar.svelte (hunk 3 above)
Comment (dylansison) [reply src/lib/Navbar.svelte]: removed

File: src/app.html
Comment (dylansison) [reply src/app.html:1]: I tried, +layout.svelte not found; using example format as presented in reference article

**Review by willuhmjs — COMMENTED** [review]
Let Tristan deal with this PR

**Comment by tristan-f-r** [comment]
This PR is stale.

**Pull request closed without merging**